soapui-testserver-api/src/main/java/com/smartbear/ready/recipe/TestStepNames.java

```diff
@@ -12,4 +14,20 @@
     public static final String DELAY_TYPE = "Delay";
     public static final String PROPERTIES_TYPE = "Properties";
     public static final String SOAP_MOCK_RESPONSE_TYPE = "SOAPMockResponse";
+
+    public static String createUniqueName(WsdlTestCase testCase, String testStepName) {
+        if (testCase.getTestStepByName(testStepName) == null) {
+            return testStepName;
+        }
+
+        String orgName = testStepName;
+        int cnt = 1;
+
+        while (testCase.getTestStepByName(testStepName) != null) {
+            testStepName = orgName + " " + String.valueOf(cnt);
```

Comment:
So if you ask for three test steps named "Smurf", you will get "Smurf", "Smurf 1" and "Smurf 2", I suppose. Shouldn't the count start at 2?